### PR TITLE
Migrate to card based langding page.

### PR DESCRIPTION
**Pull request by wingtofree**
Finish [DevOps Task](https://dev.azure.com/ceapex/Engineering/_workitems/edit/218412)
Following this guide: [Create or update a landing page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-landing-page?branch=master)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[cfe5a69](https://github.com/wingtofree/office-js-docs-pr/commits/cfe5a6973bf08c50ca081817bae3db7f3c13484a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/convert-javascript-to-typescript.md) | :warning:Warning | | [Details](#user-content-f6502b1a-5f8db941)
[docs/develop/develop-add-ins-vscode.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/develop-add-ins-vscode.md) | :warning:Warning | | [Details](#user-content-609cdd06-5f8db941)
[docs/overview/office-add-ins-fundamentals.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/office-add-ins-fundamentals.md) | :warning:Warning | | [Details](#user-content-de64f43a-5f8db941)
[docs/overview/set-up-your-dev-environment.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/set-up-your-dev-environment.md) | :warning:Warning | | [Details](#user-content-5dda2249-5f8db941)
[docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/index.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="f6502b1a-5f8db941"></a>
### [docs/develop/convert-javascript-to-typescript.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/convert-javascript-to-typescript.md)
 - **Line 13**: **[Warning]** `````Invalid file link:(~/index.md).`````

<a id="609cdd06-5f8db941"></a>
### [docs/develop/develop-add-ins-vscode.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/develop-add-ins-vscode.md)
 - **Line 25**: **[Warning]** `````Invalid file link:(~/index.md).`````
 - **Line 34**: **[Warning]** `````Invalid file link:(~/index.md).`````

<a id="de64f43a-5f8db941"></a>
### [docs/overview/office-add-ins-fundamentals.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/office-add-ins-fundamentals.md)
 - **Line 117**: **[Warning]** `````Invalid file link:(~/index.md).`````
 - **Line 117**: **[Warning]** `````Invalid file link:(~/index.md).`````

<a id="5dda2249-5f8db941"></a>
### [docs/overview/set-up-your-dev-environment.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/set-up-your-dev-environment.md)
 - **Line 61**: **[Warning]** `````Invalid file link:(~/index.md).`````
 - **Line 61**: **[Warning]** `````Invalid file link:(~/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005210327468005-1877%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Loz%2BKRthScPEwS0dwmf7aVyvZHGmYygn1RFZ0fyJ%2BVg%3D&st=2020-05-21T03%3A24%3A42Z&se=2020-06-21T03%3A29%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b137db5](https://github.com/wingtofree/office-js-docs-pr/commits/b137db59488a68d5ae38d6027d7163642952ccc7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-add-ins-vscode.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/develop-add-ins-vscode.md) | :white_check_mark:Succeeded | |
[docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/index.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/index.yml) | :white_check_mark:Succeeded | |
[docs/overview/office-add-ins-fundamentals.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/office-add-ins-fundamentals.md) | :white_check_mark:Succeeded | |
[docs/overview/set-up-your-dev-environment.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/set-up-your-dev-environment.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005210351554960-1877%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cLQQYXpAS8s9rBbLuGTm7aC98cPB4VWj45eIdHc37Cw%3D&st=2020-05-21T03%3A48%3A56Z&se=2020-06-21T03%3A53%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ElizabethSamuel-MSFT** [comment]
Testing if fork now generates preview site.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b137db5](https://github.com/wingtofree/office-js-docs-pr/commits/b137db59488a68d5ae38d6027d7163642952ccc7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/convert-javascript-to-typescript.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/convert-javascript-to-typescript.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/convert-javascript-to-typescript?branch=pr-en-us-1877) |
[docs/develop/develop-add-ins-vscode.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/develop/develop-add-ins-vscode.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/develop-add-ins-vscode?branch=pr-en-us-1877) |
[docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/index.md) | :white_check_mark:Succeeded | |
[docs/index.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-1877) |
[docs/overview/office-add-ins-fundamentals.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/office-add-ins-fundamentals.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins-fundamentals?branch=pr-en-us-1877) |
[docs/overview/set-up-your-dev-environment.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/overview/set-up-your-dev-environment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/set-up-your-dev-environment?branch=pr-en-us-1877) |
[docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-langding/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins-fundamentals?branch=pr-en-us-1877) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006081617339500-1877%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=mCVtQhCBWXKv9B1u3xBcLo0pIleGTg9XFD8YWIx6bU4%3D&st=2020-06-08T16%3A14%3A22Z&se=2020-07-09T16%3A19%3A22Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/index.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+summary: Learn how to build Office Add-ins. # < 160 chars
+
+metadata:
+  title: Office Add-ins documentation # Required; page title displayed in search results. Include the brand. < 60 chars.
+  description: Learn how to build Office Add-ins. # Required; article description that is displayed in search results. < 160 chars.
+  ms.service: o365-proplus-itpro #Required; service per approved list. service slug assigned to your service by ACOM.
+  ms.topic: landing-page # Required
+  author: o365devx #Required; your GitHub user alias, with correct capitalization.
+  ms.author: o365devx #Required; microsoft alias of author; optional team alias.
+  ms.date: 12/24/2019 #Required; mm/dd/yyyy format.
```
Comment (Rick-Kirkham) [inline docs/index.yml:13]: update ms.date

**Pull request merged**